SDK/Hardware/HA4IoT.Hardware.Knx/KnxDigitalJoinEnpoint.cs

```diff
@@ -0,0 +1,51 @@
+﻿using HA4IoT.Contracts.Actuators;
+using HA4IoT.Contracts.Hardware;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace HA4IoT.Hardware.Knx
+{
+    public class KnxDigitalJoinEnpoint : IBinaryStateEndpoint
+    {
+        private readonly string _identifier;
+        private readonly KnxController _knxController;
+
+        public KnxDigitalJoinEnpoint(string identifier, KnxController knxController)
+        {
+            if (identifier == null) throw new ArgumentNullException(nameof(identifier));
+            if (knxController == null) throw new ArgumentNullException(nameof(knxController));
+
+            if (ValidationJoin(identifier))
+            {
+                _identifier = identifier;
+                _knxController = knxController;
+            }
```

Comment:
I think you forgot the "else" part with an ArgumentException? If there is no code the call to TurnOn or TurnOff will crash because both fields are null (NullReferenceException)